build-tasks/markup.js

```diff
@@ -13,9 +13,11 @@ var gulp      = require('gulp'),
   },
   /* markup:compile */
   compile = function() {
+    if (env.deploy && opts.pug.pretty) opts.pug.pretty = false;
+
     return gulp.src(src.docs)
       .pipe(plugins.plumber())
-      .pipe(env.deploy ? plugins.pug(): plugins.pug(opts.pug))
+      .pipe(turnOffPrettyPrint())
```

Comment:
The function `turnOffPrettyPrint` is still present.

This just needs to be `plugins.pug(opts.pug)` 👍